docs/tempo/website/guides/instrumentation.md

```diff
@@ -2,8 +2,15 @@
 title: Instrumentation Examples
 ---
 
-Check out these blog posts for help instrumenting your favorite languages to work with Tempo.
+Check out these resources for help instrumenting your favorite languages to work with Tempo.
```

Comment:
I think we should add a link or mention this docs on the [OpenTelemetry section](https://github.com/grafana/tempo#opentelemetry) of the readme.